src/components/TimeGrid.jsx

```diff
@@ -0,0 +1,239 @@
+          || !dates.eq(eStart, eEnd, 'day')
+          || (dates.isJustDate(eStart) && dates.isJustDate(eEnd)))
+        {
+          allDayEvents.push(event)
+        }
+        else
+          rangeEvents.push(event)
+      }
+    })
+    return {
+      allDayEvents,
+      rangeEvents
+    }
+  }
+
+  renderEvents(range, events){
+    let { min, max, endAccessor, startAccessor, components } = this.props;
+    let today = new Date(), daysEvents=[]
+
+    return range.map((date, idx) => {
+       let daysEvents = events.filter(
+         event => dates.inRange(date,
+           get(event, startAccessor),
+           get(event, endAccessor), 'day')
+       )
+
+      return (
+        <DaySlot
+          {...this.props }
+          min={dates.merge(date, min)}
+          max={dates.merge(date, max)}
+          now={new Date()}
+          eventTimeRangeFormat={this.props.eventTimeRangeFormat}
+          formatter={({start, end}) => {
+            if (start+'' === end+'') {
+              return localizer.format(
+                {start, end: dates.add(end, this.props.step, 'minutes')},
+                this.props.selectRangeFormat,
+                this.props.culture)
+            }
+            return localizer.format(
+              {start, end},
+              this.props.selectRangeFormat,
+              this.props.culture)
+          }}
+          className={cn({ 'rbc-now': dates.eq(date, today, 'day') })}
+          style={segStyle(1, range.length)}
+          key={idx}
+          date={date}
+          events={daysEvents}
+        />
+      )
+    })
+  }
+
+  renderAllDayEvents(allDayEvents, range){
+    const events = allDayEvents.slice(0)
+
+    events.sort((a, b) => sortEvents(a, b, this.props))
+
+    const { first, last } = endOfRange(range);
+    const segments = events.map(evt => eventSegments(evt, first, last, this.props))
+    const { levels } = eventLevels(segments)
+
+    while (levels.length < MIN_ROWS )
+      levels.push([])
+
+    return levels.map((segs, idx) =>
+      <EventRow
+        eventComponent={this.props.components.event}
+        titleAccessor={this.props.titleAccessor}
+        startAccessor={this.props.startAccessor}
+        endAccessor={this.props.endAccessor}
+        allDayAccessor={this.props.allDayAccessor}
+        eventPropGetter={this.props.eventPropGetter}
+        onSelect={() => null}
+        slots={range.length}
+        key={idx}
+        segments={segs}
+        start={first}
+        end={last}
+      />
+    )
+  }
+
+  render() {
+    const addGutterRef = i => ref => this._gutters[i] = ref
+
+    const range = dates.range(this.props.start, this.props.end, 'day')
+    const { allDayEvents, rangeEvents } = this.separateEvents()
+
+    return (
+      <div className='rbc-time-view'>
+        <div className="rbc-time-header" ref={ref => this._headerCell = ref}>
+          <TimeGridHeader range={range}
+                          gutterRef={addGutterRef(0)}
```

Comment:
fyi (not a big deal), sorry by align I ment like he components above in renderAllDayEvents, props should be indented once and all on a new line under the tag, not aligned to the first prop. Just for going forward